aspnetcore/tutorials/getting-started-with-swashbuckle.md

```diff
@@ -3,15 +3,15 @@ title: Get started with Swashbuckle and ASP.NET Core
 author: zuckerthoben
 description: Learn how to add Swashbuckle to your ASP.NET Core web API project to integrate the Swagger UI.
 ms.author: scaddie
-monikerRange: '>= aspnetcore-3.1 < aspnetcore-8.0'
+monikerRange: '>= aspnetcore-3.1'
 ms.custom: mvc
 ms.date: 11/09/2021
 uid: tutorials/get-started-with-swashbuckle
 ---
 # Get started with Swashbuckle and ASP.NET Core
 
 > [!NOTE]
-> Build-time OpenAPI document generation with Swashbuckle is not supported in .NET 8 and later. For a supported alternative, see <xref:tutorials/web-api-help-pages-using-swagger>.
+> Build-time OpenAPI document generation with Swashbuckle is not supported in .NET 8 and later. For a supported  build-time  alternative, see <xref:tutorials/web-api-help-pages-using-swagger>. Run-time support is not impacted.
```

Comment:
@captainsafia The original was off putting. I'm hoping to tone it down, closer to the actual limitations.

Reply:
This change looks good!